tests/unit/htdocs/class/XoopsMediaUploaderTest.php

```
@@ -0,0 +1,809 @@
+        $ref = new ReflectionClass($uploader);
+        $prop = $ref->getProperty('maxFileSize');
+        $prop->setAccessible(true);
+
+        // Use unset trick via reflection on a fresh object that never had it set
+        $fresh = $ref->newInstanceWithoutConstructor();
+        // maxFileSize is declared as public $maxFileSize = 0 in the class,
+        // so it will always be "set". The check is `!isset($this->maxFileSize)`.
+        // With value 0, isset() returns true (0 is set). But the method checks !isset.
+        // A default-initialized property with value 0 IS set in PHP.
+        // So we need to genuinely unset it. We can do this via cast.
+        $arr = (array) $fresh;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$arr'. (undefined)

(UnusedLocalVariable)